DigitalLearningSolutions.Web/Views/SuperAdmin/Centres/EditContractInfo.cshtml

```diff
@@ -0,0 +1,78 @@
+﻿@using DigitalLearningSolutions.Web.ViewModels.SuperAdmin.Centres
+@model ContractTypeViewModel
+@{
+    var errorHasOccurred = !ViewData.ModelState.IsValid;
+    ViewData["Title"] = errorHasOccurred ? "Error: Edit Contract Info" : "Edit Contract Info";
+ 
+}
+
+@if (errorHasOccurred)
+{
+    <vc:error-summary order-of-property-names="@(new string[] { nameof(Model.CentreName),nameof(Model.ContractType)})" />
+}
+ 
+
+<div id="form-group">
+    <h1 class="nhsuk-heading-xl">Edit contract info for @Model.CentreName </h1>
+    <form method="post">
+     
+                    @Html.HiddenFor(x=>x.CentreId)
+                    @Html.HiddenFor(x=>x.CentreName)
+                    @Html.HiddenFor(x=>x.ContractType)
+
+                    <div class="nhsuk-grid-row divider">
+                        <div class="nhsuk-grid-column-full">
+                            <vc:select-list asp-for="@nameof(Model.ContractTypeID)"
+                                            label="Contract type"
+                                value="@Model.ContractType.ToString()"
+                                           hint-text=""
+                                            required="true"
+                                            css-class="nhsuk-u-width-one-half"
+                                            default-option=""
+                                            select-list-options="@ViewBag.ContractTypes" />
+
+                            <vc:select-list asp-for="@nameof(Model.ServerSpaceBytesInc)"
+                                            label="Server space"
+                                            value="@Model.ServerSpaceBytesInc.ToString()"
+                                            hint-text=""
+                                            required="true"
+                                            css-class="nhsuk-u-width-one-half"
+                                            default-option=""
+                                            select-list-options="@ViewBag.Serverspace" />
+
+
+                            <vc:select-list asp-for="@nameof(Model.DelegateUploadSpace)"
+                                            label=" Per delegate upload space"
+                                            value="@Model.DelegateUploadSpace"
+                                            hint-text=""
+                                            required="true"
+                                            css-class="nhsuk-u-width-one-half"
+                                            default-option=""
+                                            select-list-options="@ViewBag.Delegatespace" />
+                  <div class="nhsuk-form-group">
+                    <label class="nhsuk-label" for="ContractReviewDate">
+                    Contract Review Date
+                </label>
+                    <input type="date" id="ContractReviewDate" name="ContractReviewDate" class="nhsuk-input nhsuk-u-width-one-half" asp-for="ContractReviewDate" />
```

Comment:
Change this date field to use the standard date control. Please refer to other areas of the product like register user journey or follow the link https://design-system.service.gov.uk/components/date-input/